packages/select/src/vaadin-select.js

```diff
@@ -326,6 +340,8 @@ class Select extends OverlayClassMixin(
     super();
 
     this._itemId = `value-${this.localName}-${generateUniqueId()}`;
+    this._srLabelController = new LabelController(this);
+    this._srLabelController.slotName = 'sr-label';
```

Comment:
`LabelController` doesn't allow the definition of any custom slot name on its constructor. The easiest approach is to set the `slotName` property immediately after its instantiation. Another approach is to update `LabelController` constructor to something like:
```js
constructor(host, slotName = 'label') {
	super(host, slotName, 'label');
}
```